cime_config/testdefs/ExpectedTestFails.xml

```diff
@@ -12,7 +12,7 @@
   </category>
   <category name="fates">
     <entry issue="NGEET/fates/#559" >FAIL SMS_Lm3_D_Mmpi-serial.1x1_brazil.I2000Clm50FatesCruGs.cheyenne_intel.clm-FatesHydro RUN</entry>
-    <entry issue="667"              >FAIL ERS_D_Ld5.1x1_brazil.I2000Clm50FatesCruGs.cheyenne_intel.clm-FatesHydro COMPARE_base_rest</entry>
+    <entry issue="991"              >FAIL ERS_D_Lm12.1x1_brazil.I2000Clm50FatesCruGs.cheyenne_intel.clm-FatesSuccIgnitions COMPARE_base_rest</entry>
```

Comment:
The tests referencing issue #667 seem fixed. I did not remove the rest of them from this list in case I am mistaken.

However, the **COMPARE_base_rest** failure reappears now with this new test. I have confirmed that the failure also occurs with `fates_spitfire_mode = 1` which suggests to me a preexisting problem.

Reply:
I further confirmed that the **COMPARE_base_rest** failure is a preexisting problem by switching to fates_next_api and master and submitting this test:
ERS_D_Lm12.1x1_brazil.I2000Clm50FatesCruGs.cheyenne_intel.clm-Fates

@glemieux @ekluzek would you like me to open an issue?

Reply:
@slevisconsulting yes please open an issue.  I'm not quite familiar with the background so detailing it in an issue may help me.